### DISCUSSION TITLE
Does kitty support background-mode in the same manner as KDE Konsole?

### DISCUSSION THREAD

**Discussion by gbcox** [kovidgoyal/kitty#5725] [General]
Hi, I searched for this and found where you can open a session in the background without it being associated with a terminal, but I couldn't find how to duplicate the functionality of Konsole, where you can open a terminal session in the background, running a script and then use a hotkey or script command to bring it back and forth to the background.  Just wondering if this is possible in kitty?  Thanks!

**Comment by kovidgoyal** [comment]
kstart --iconify kitty whatever

As for hwo to bring it to the fron click on it in the taskbar or assign
a shorcut for it in kwin.

**Comment by gbcox** [comment]
Thank you very much for the quick reply, much appreciated!
